public/docs/ts/latest/quickstart.jade

```diff
@@ -60,13 +55,9 @@
 .l-main-section
   h2#section-transpile 3. Import Angular
 
-  p Inside of <code>app.ts</code>, import the type definitions from Angular:
+  p Inside of <code>app.ts</code>, your editor should be able to complete the available imports:
```

Comment:
dont we also need to say the module system? when indicating systemjs it has failed often for me. when using commonjs it works.